elastic/kibana
# [Infrastructure UI] Track host cloud provider on table entry click

## 📓 Summary

Closes #150681 

The tracked event when clicking an entry from the Hosts table was not applying correctly the default `unknow` string for the `cloud_provider` parameter.
Now it defaults to it in case the value is not passed or is null.

## 🧪 Testing
- Navigate to the Hosts view
- Open the console dev tools
- Click on a host name from the table and navigate to the host detail.
- Verify the following error does not appear in the log
```
Uncaught Error: Failed to validate payload coming from "Event Type 'Host Entry Clicked'":
[cloud_provider]: {"expected":"string","actual":"object","value":null}
```
